### ISSUE TITLE
configure file missing in 3.0.721

### ISSUE THREAD

**Issue by uniuuu** [emikulic/darkstat#21]
Hi @emikulic 

As per INSTALL:
```
darkstat-3.0.721]# more INSTALL 
Installation instructions
-------------------------

$ ./configure
$ make
$ make install

```

But in the latest release 3.0.721 in the tarball there is no configure file included  https://github.com/emikulic/darkstat/archive/refs/tags/3.0.721.tar.gz 

I was able to generate `configure` with `autoreconf -i` though. 

```
darkstat --version
darkstat 3.0.721 (using libpcap version 1.9.1 (with TPACKET_V3))
```

**Comment by emikulic** [comment]
Yep. 721 was the first "release" to just use a github link. Previously I would use the release script to generate a tarball with a configure script etc: https://github.com/emikulic/darkstat/blob/master/release.sh

I'm not sure what the best approach is going forward.

**Comment by emikulic** [comment]
Explicitly generating a release tarball via e.g. GitHub actions would solve this problem, and also the problem where GitHub "archive" links aren't guaranteed to be stable over time (i.e. to have the same hash)

**Comment by piyushgarg** [comment]
I was able to build it by the instructions of @uniuuu. @emikulic Can you generate an configure file using the above instruction and check that in?

**Comment by emikulic** [comment]
I'm sorry but I don't want to check in generated artifacts. Maybe I should just update the INSTALL instructions.

**Comment by cmeury** [comment]
Potentially add the required libaries as well, for Debian-based systems:

```shell
sudo apt-get install automake libpcap-dev
```

**Comment by emikulic** [comment]
Kinda documented later in the file: https://github.com/emikulic/darkstat/blob/a219027ec8ea6bb75647bef126dd44f56ac9c86e/INSTALL#L47

**Comment by emikulic** [comment]
> I was able to build it by the instructions of @uniuuu. @emikulic Can you generate an configure file using the above instruction and check that in?

Turns out this is documented in https://github.com/emikulic/darkstat/blob/master/README.git

Maybe there's too much documentation.
